front/app/containers/Admin/projects/project/analysis/TopBar/Filters/AuthorFilters.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { useMemo } from 'react';
+import useUserCustomFields from 'api/user_custom_fields/useUserCustomFields';
+import useUserCustomFieldsOptions from 'api/user_custom_fields_options/useUserCustomFieldsOptions';
+import { Box, Label, Select } from '@citizenlab/cl2-component-library';
+import useLocalize from 'hooks/useLocalize';
+import MultipleSelect from 'components/UI/MultipleSelect';
+import { updateSearchParams } from 'utils/cl-router/updateSearchParams';
+import { useSearchParams } from 'react-router-dom';
+import { useIntl } from 'utils/cl-intl';
+import messages from '../messages';
+
+const AuthorFilters = () => {
+  const localize = useLocalize();
+  const { formatMessage } = useIntl();
+  const [searchParams] = useSearchParams();
+  const { data: customFields } = useUserCustomFields();
+
+  const genderField = customFields?.data.find(
+    (field) => field.attributes.code === 'gender'
+  );
+
+  const domicileField = customFields?.data.find(
+    (field) => field.attributes.code === 'domicile'
+  );
+
+  const birthyearField = customFields?.data.find(
+    (field) => field.attributes.code === 'birthyear'
+  );
+
+  const { data: genderOptions } = useUserCustomFieldsOptions(genderField?.id);
+  const { data: domicileOptions } = useUserCustomFieldsOptions(
+    domicileField?.id
+  );
+
+  const genderUrlQueryParamKey = `author_custom_${genderField?.id}`;
+  const domicileUrlQueryParamKey = `author_custom_${domicileField?.id}`;
+  const birthyearUrlQueryParamFromKey = `author_custom_from_${birthyearField?.id}`;
+  const birthyearUrlQueryParamToKey = `author_custom_to_${birthyearField?.id}`;
```

Comment:
These differ from [the spec](https://www.notion.so/citizenlab/Shaping-sensemaking-f9d1c238a7014391839d6a56e4a79b55?pvs=4#41dfe69e2f6341cebbbc02a4cb79d155), guess you did that on purpose? Makes more sense to me to have it in the other way (first field, than predicate). Either way, easy change on either side.

Reply:
Indeed, wasn't on purpose. I'll change this.